db/re/item_db.txt

```diff
@@ -11362,6 +11365,13 @@
 //===================================================================
 // New etc.
 //===================================================================
+25044,Hard_Thorny_Vine,Hard Thorny Vine,3,6,,10,,,,,,,,,,,,,{},{},{}
+25043,Thorny_Vine_Flute,Thorny Vine Flute,3,6,,10,,,,,,,,,,,,,{},{},{}
```

Comment:
Needs to be added to /db/re/item_trade.txt:
`25043,499,100	// Thorny_Vine_Flute`